src/Catty/DataModel/Look/Look.swift

```diff
@@ -0,0 +1,76 @@
+/**
+ *  Copyright (C) 2010-2020 The Catrobat Team
+ *  (http://developer.catrobat.org/credits)
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  An additional term exception under section 7 of the GNU Affero
+ *  General Public License, version 3, is available at
+ *  (http://developer.catrobat.org/license_additional_term)
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see http://www.gnu.org/licenses/.
+ */
+@objc(Look)
+@objcMembers class Look: NSObject, CBMutableCopying {
+
+    var fileName: String
+    var name: String
+
+    @objc init(withName name: String, andPath filePath: String) {
```

Comment:
`withName...andPath` sounds great for ObjC (you can explicitly define this name for ObjC only), but for Swift please use the initializer without `withName` and `andPath` (since the arguments correspond directly to the stored properties).